src/main/java/gregtech/loaders/recipe/WoodMachineRecipes.java

```diff
@@ -38,36 +39,37 @@ private static void processLogOreDictionary() {
         for (ItemStack stack : allWoodLogs) {
             Pair<IRecipe, ItemStack> outputPair = ModHandler.getRecipeOutput(null, stack);
             ItemStack plankStack = outputPair.getValue();
-            int originalOutput = plankStack.getCount();
             if (plankStack.isEmpty()) {
                 continue;
             }
             IRecipe outputRecipe = outputPair.getKey();
 
             //wood nerf
-            if (ConfigHolder.recipes.nerfWoodCrafting) {
+            if (ConfigHolder.recipes.WoodCraftingNumber != 4) {
                 //remove the old recipe
                 ModHandler.removeRecipeByName(outputRecipe.getRegistryName());
 
                 // new wood recipes
                 //noinspection ConstantConditions
-                ModHandler.addShapelessRecipe(outputRecipe.getRegistryName().toString(),
-                        GTUtility.copyAmount(Math.max(1, originalOutput / 2), plankStack), stack);
+                if (ConfigHolder.recipes.WoodCraftingNumber > 0) {
+                    ModHandler.addShapelessRecipe(outputRecipe.getRegistryName().toString(),
+                            GTUtility.copyAmount(ConfigHolder.recipes.WoodCraftingNumber, plankStack), stack);
 
-                ModHandler.addShapedRecipe(outputRecipe.getRegistryName().getPath() + "_saw",
-                        GTUtility.copyAmount(originalOutput, plankStack), "s", "L", 'L', stack);
+                    ModHandler.addShapedRecipe(outputRecipe.getRegistryName().getPath() + "_saw",
+                            GTUtility.copyAmount((int) (ConfigHolder.recipes.WoodCraftingNumber * 1.5), plankStack), "s", "L", 'L', stack);
```

Comment:
This is a behavior change, as before the saw would not give bonus output if the wood recipes were nerfed, now the saw always gives bonus output.